apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx

```diff
@@ -77,6 +78,13 @@ export function ValidatorFormDetail({ validatorAddress, unstake }: ValidatorForm
                         value={formatPercentageDisplay(apy, '--', isApyApproxZero)}
                         fullwidth
                     />
+                    <KeyValueInfo
+                        keyText="Validator commission"
+                        value={commission}
+                        fullwidth
+                        tooltipText="The charge imposed by the validator for their staking services."
+                        tooltipPosition={TooltipPosition.Right}
```

Comment:
When you set the position on the right, the tooltip is cut off
![image](https://github.com/user-attachments/assets/84cfbdf9-7bb3-44aa-9989-9b374aa4e763)
```suggestion
                        tooltipPosition={TooltipPosition.Top}
```